sdk/cosmos/azure_data_cosmos/src/constants.rs

```diff
@@ -9,8 +9,6 @@ use azure_core::{headers::HeaderName, request_options::ContentType};
 pub const QUERY: HeaderName = HeaderName::from_static("x-ms-documentdb-query");
 pub const PARTITION_KEY: HeaderName = HeaderName::from_static("x-ms-documentdb-partitionkey");
 pub const CONTINUATION: HeaderName = HeaderName::from_static("x-ms-continuation");
-pub const INDEX_METRICS: HeaderName = HeaderName::from_static("x-ms-cosmos-index-utilization");
-pub const QUERY_METRICS: HeaderName = HeaderName::from_static("x-ms-documentdb-query-metrics");
```

Comment:
I'll replace these with typed headers (#1829) later.

Reply:
Granted, but any point in deleting them now? Or maybe open a tracking issue?

Reply:
They're unused, because currently the constants module is `pub(crate)`. I suppose that module can be public too though.